pyccl/ccl_core.i

```diff
@@ -35,50 +57,4 @@ void parameters_get_nu_masses(ccl_parameters *params, int nout, double* output)
     }
 }
 
-ccl_parameters parameters_create_nu(
-    double Omega_c, double Omega_b, double Omega_k, double Neff,
-    double w0, double wa, double h, double A_s, double sigma8, double n_s,
-    double T_CMB, double Omega_g, double T_ncdm,
-    double bcm_log10Mc, double bcm_etab, double bcm_ks, double mu_0,
-    double sigma_0, double c1_mg, double c2_mg, double lambda_mg,
-    double* m_nu, int n_m, int* status)
-{
-    return ccl_parameters_create(
-        Omega_c, Omega_b, Omega_k, Neff, m_nu, n_m, w0, wa, h,
-        A_s, sigma8, n_s, T_CMB, Omega_g, T_ncdm,
-        bcm_log10Mc, bcm_etab, bcm_ks,
-        mu_0, sigma_0, c1_mg, c2_mg, lambda_mg,
-        -1, NULL, NULL, status );
-}
-
-%}
-
-%feature("pythonprepend") parameters_create_nu_vec %{
-    if numpy.shape(zarr) != numpy.shape(dfarr):
-        raise CCLError("Input shape for `zarr` must match `dfarr`!")
-%}
-
-%inline %{
-ccl_parameters parameters_create_nu_vec(
-    double Omega_c, double Omega_b, double Omega_k, double Neff,
-    double w0, double wa, double h, double A_s, double sigma8, double n_s,
-    double T_CMB, double Omega_g, double T_ncdm,
-    double bcm_log10Mc, double bcm_etab, double bcm_ks, double mu_0,
-    double sigma_0, double c1_mg, double c2_mg, double lambda_mg,
-    double* zarr, int nz,
-    double* dfarr, int nf, double* m_nu,
-    int n_m, int* status)
-{
-    if (nz == 0){ nz = -1; }
-    return ccl_parameters_create(
-        Omega_c, Omega_b, Omega_k, Neff, m_nu, n_m, w0, wa, h,
-        A_s, sigma8, n_s, T_CMB, Omega_g, T_ncdm,
-        bcm_log10Mc, bcm_etab, bcm_ks,
-        mu_0, sigma_0, c1_mg, c2_mg, lambda_mg,
-        nz, zarr, dfarr, status);
-}
```

Comment:
Is `ccl_parameters_create` ever being called now?

Reply:
Yes, only at the C-level by MG stuff and halofit. Otherwise I'd remove it.

Reply:
I see, OK

Reply:
Oh, hold on, but those will rely on the default values of T_CMB and T_ncdm, which now live in python, right? I know there are copies at the C level, but it'd be good if we can roadblock them completely. See a global comment further down.